core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -90,7 +90,7 @@ case class WorkspaceBillingAccount(
   currentBillingAccountOnGoogleProject: Option[RawlsBillingAccountName]
 )
 
-case class RawlsBillingProjectOrganization(enterprise: Boolean)
+case class RawlsBillingProjectOrganization(enterprise: Boolean, limits: Map[String, String])
```

Comment:
will the `limits` key always be present in the BPM response, even if its value is `{}`? If not, you may need to make this an `Option[Map[String,String]]` to deal with the missingness

Reply:
Yes, I managed to have it default to an empty map in BPM, so if there's no limits present it will be {}